### PR TITLE
Add security.txt

### PR DESCRIPTION
**Pull request by taskinoz**
Added a `security.txt` file to the `.well-known` directory so people know where to report security issues

### CODE REVIEW

File: .well-known/security.txt [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+# Northstar Launcher
+https://github.com/R2Northstar/NorthstarLauncher
+
+# Northstar Game Scripts
+https://github.com/R2Northstar/NorthstarMods
+
+# Or notrify one of our developers on Discord
+Gecko#7945
+EmmaM#6474
```
Comment (GeckoEidechse) [inline .well-known/security.txt:16]: With d874cc07f58011a52f6ecd0f692e935242b1f504 this file should now be moved to `dist/.well-known/security.txt`

**Review by GeckoEidechse — CHANGES_REQUESTED** [review]

File: .well-known/security.txt [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+# Northstar Atlas (Master Server)
+https://github.com/R2Northstar/Atlas
+
+# Northstar Launcher
+https://github.com/R2Northstar/NorthstarLauncher
+
+# Northstar Game Scripts
+https://github.com/R2Northstar/NorthstarMods
+
+# Or notrify one of our developers on Discord
```
Comment (pg9182) [inline .well-known/security.txt:14]: `notrify`

Comment (taskinoz) [reply .well-known/security.txt]: How did I miss this twice -_-

Updated

File: .well-known/security.txt (hunk 1 above)
Comment (taskinoz) [reply .well-known/security.txt]: Updated

File: dist/.well-known/security.txt [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+# Northstar Github page
+
+# Northstar Atlas (Master Server)
+https://github.com/R2Northstar/Atlas
+
+# Northstar Launcher
+https://github.com/R2Northstar/NorthstarLauncher
+
+# Northstar Game Scripts
+https://github.com/R2Northstar/NorthstarMods
```
Comment (GeckoEidechse) [inline dist/.well-known/security.txt:12]: Looking at again, maybe we should also link the org as a whole? ^^

Comment (taskinoz) [reply dist/.well-known/security.txt:12]: So just https://github.com/R2Northstar?

Comment (taskinoz) [reply dist/.well-known/security.txt:15]: Updated

**Review by GeckoEidechse — APPROVED** [review]
Looks good to me ^^

**Pull request merged**